vpr/src/base/read_options.cpp

```diff
@@ -1591,6 +1591,13 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("64")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    route_timing_grp.add_argument<float>(args.router_high_fanout_max_slope, "--router_high_fanout_max_slope")
+        .help(
+            "Maximum routing predictor slope where high fanout routing is enabled.\n"
+            " -1.0 is normal progress, 0 is no progress.")
+        .default_value("-0.5")
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
```

Comment:
I think this description is fairly opaque and needs some elaboration.

What is the slope being considered? What are its units? Why is -1 normal progress, and the default set to -0.5? What value would disable this feature (e.g. may want to disable it during minW search if the run-time hit is high)?

Ideally we want a tunable which is somewhat intuitive for those who don't know the internals of the routing predictor.

Perhaps something like an expected routing convergence iteration threshold would be more intuitive. For example, disable HF routing if: `predicted_success_iteration / max_router_iterations > alpha` (where alpha is the parameter this option controls).

Reply:
I've updated the description.